src/test/kotlin/domain/CarsTest.kt

```diff
@@ -0,0 +1,35 @@
+package domain
+
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+
+private class FixedMoveDecision(private val shouldMove: Boolean) : MoveDecision {
+    override fun shouldMove(): Boolean = shouldMove
+}
+
+class CarsTest : BehaviorSpec({
+
+    Given("여러 대의 자동차가 있고") {
+        val car1 = Car()
+        val car2 = Car()
+        val car3 = Car()
+
+        When("모든 자동차가 전진하도록 결정된 경우") {
+            val cars = Cars(listOf(car1, car2, car3), FixedMoveDecision(true))
```

Comment:
[Functional interface](https://kotlinlang.org/docs/fun-interfaces.html)를 활용해주신 만큼, 다음과 같이 작성해볼 수도 있을 것 같아요. (Kotlin 달달하시죠? 😆 )
```suggestion
            val cars = Cars(listOf(car1, car2, car3), MoveDecision { true })
```

Reply:
![짱구 초롱](https://github.com/user-attachments/assets/09d3774f-d9fa-416e-8d16-e37bbae55315)
<img width="305" alt="image" src="https://github.com/user-attachments/assets/085c02f2-d76c-4d95-9c24-777183b5b26d">

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 아이고 빵터졌네요